packages/components/src/List/Toolbar/ColumnChooserButton/ColumnChooser/ColumnChooserHeader/ColumnChooserHeader.component.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useColumnChooserContext } from '../columnChooser.context';
+import Tooltip from '../../../../../Tooltip';
+import cssModule from '../ColumnChooser.scss';
+import { getTheme } from '../../../../../theme';
+
+const theme = getTheme(cssModule);
+
+const filterVisibleColumns = column => column.visible;
```

Comment:
```suggestion
const isVisible = column => column.visible;
```
So you can write `columns.filter(isVisible)` instead of `columns.filter(filterVisibleColumns)`